src/component/TitleBar/TitleBar.scss

```diff
@@ -16,18 +16,20 @@
         position: absolute;
         margin-top: -27px;
         border: 0px solid #c14c46;
+        border-top-left-radius: 7px;
+        border-top-right-radius: 7px;
```

Comment:
This needs to be conditional since you are breaking title bars for any action after the first one that is being removed.